packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/browser/ModelBrowsersDescriptionProvider.java

```diff
@@ -397,4 +398,13 @@ private IStatus getDeleteHandler(VariableManager variableManager) {
     private IStatus getRenameHandler(VariableManager variableManager, String newLabel) {
         return new Failure("");
     }
+
+    private Object getObject(VariableManager variableManager) {
```

Comment:
`getTreeItemObject` to be consistent with other methods

Reply:
Done